src/java.base/share/classes/java/security/spec/PSSParameterSpec.java

```diff
@@ -64,13 +64,18 @@
  *   ...  -- Allows for future expansion --
  * }
  * </pre>
- * <p>Note: the PSSParameterSpec.DEFAULT uses the following:
+ * <p>Note: the {@code PSSParameterSpec.DEFAULT} uses the following:
  *     message digest  -- "SHA-1"
  *     mask generation function (mgf) -- "MGF1"
  *     parameters for mgf -- MGF1ParameterSpec.SHA1
  *     SaltLength   -- 20
  *     TrailerField -- 1
  *
+ * <p>Its values are based on the default values defined in the ASN.1 encoding
+ * from the PKCS#1 standard and may become obsolete as time progresses.
+ * Please do not rely on {@code PSSParameterSpec.DEFAULT} unless these
+ * values are really what you want to use.
+ *
```

Comment:
I just read the whole class spec. Since the default values are already included above on lines 38-41, I think there is no need to use so much words and we can just say "defined above".

Reply:
I am not sure if everyone reads the complete spec from class to fields and methods. Which part do you want to remove and just use "defined above"? Personally, I find "above" kind of vague... It's better to keep the comment near the field. Perhaps we should trim the class spec instead?

Reply:
Almost the whole class spec is copied from PKCS #1 but at the end you have "defined in the ASN.1 encoding from the PKCS#1 standard", which might make people wonder if it is something else. If you don't want to remove anything, how about "defined in the ASN.1 encoding from the PKCS#1 standard above". It's verbose but not confusing.

BTW, on https://docs.oracle.com/en/java/javase/18/docs/api/java.base/java/security/spec/PSSParameterSpec.html, the whole "Note" is shown on one line. I guess that's not intended.

Reply:
Ah, Ok, I think I see what you are saying now. I agree about trimming the class spec. Are you ok with removing line 67-77 (proposed/new version)? This would reduce the duplication of wordings.

Reply:
Good suggestion.